src/utils/export.ts

```diff
@@ -0,0 +1,61 @@
+import {UsersState} from "types/store";
+import {parse} from "json2csv";
+import {NoteClientModel} from "types/note";
+import {VoteClientModel} from "types/vote";
+import {BoardClientModel} from "types/board";
+import JSZip from "jszip";
+import {saveAs} from "file-saver";
+
+export type ExportProps = {
+  board: BoardClientModel;
+  notes: NoteClientModel[];
+  votes: VoteClientModel[];
+  users: UsersState;
+};
+
+export const generateCSV = (state: ExportProps) => {
+  const columns = parse(state.board.columns, {quote: "", fields: ["id", "name", "color", "hidden"]});
+  const board = parse(state.board, {
+    quote: "",
+    fields: ["id", "name", "joinConfirmationRequired", "encryptedContent", "showAuthors", "voting", "votingIteration", "showNotesOfOtherUsers", "createdAt", "updatedAt", "owner"],
+  });
+  const notes = parse(state.notes, {quote: "", fields: ["id", "columnId", "text", "author", "parentId", "createdAt", "updatedAt"]});
+  const votes = parse(state.votes, {quote: "", fields: ["id", "board", "note", "user", "votingIteration"]});
+  const users = parse(state.users.all, {quote: "", fields: ["id", "displayName", "admin", "createdAt", "updatedAt", "online"]});
+  return [columns, board, notes, votes, users];
```

Comment:
I find the concat of those lists very hard to read and to work with for excel magicians

```
id,name,color,hidden
CPYBRHGDboh4myFOjkhRFgpjnIDpEQMK,Positive,backlog-blue,false
mjN3HlupyvWNHe6kiKANVC3PVNDOIX26,Negative,lean-lilac,false
qc8SIXrOghvHz0rR1OVALWPA9qxizlF0,Actions,planning-pink,false

id,name,joinConfirmationRequired,encryptedContent,showAuthors,voting,votingIteration,showNotesOfOtherUsers,createdAt,updatedAt,owner
W9WeMagefpSG8WZsl6t9JvQfBRKzahRe,BoardName,false,false,true,disabled,0,true,2021-10-06T20:15:04.073Z,2021-10-06T20:15:04.108Z,nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz

id,columnId,text,author,parentId,createdAt,updatedAt
VBjtHDktZi0KZxRwkSlp2UHbkyPCrwFW,CPYBRHGDboh4myFOjkhRFgpjnIDpEQMK,asdfasdf,nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz,,2021-10-06T20:15:06.562Z,2021-10-06T20:15:06.562Z
9VAq7GPRu2ER3oDVFDSpu994YUosLvPR,CPYBRHGDboh4myFOjkhRFgpjnIDpEQMK,asdf,nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz,,2021-10-06T20:15:06.904Z,2021-10-06T20:15:06.904Z
Dq5nEpjs18riSXpbXZjoHw0GGnmgq6yT,CPYBRHGDboh4myFOjkhRFgpjnIDpEQMK,asdf,nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz,,2021-10-06T20:15:07.167Z,2021-10-06T20:15:07.167Z
KRoQahtJx6wGemJWHbhLR1TvAk1vi2Dm,mjN3HlupyvWNHe6kiKANVC3PVNDOIX26,asdf,nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz,,2021-10-06T20:15:08.504Z,2021-10-06T20:15:08.504Z

id,board,note,user,votingIteration

id,displayName,admin,createdAt,updatedAt,online
nnJHh2nMEPwF1wvYoqnz21sfQaIAvYJz,Naive Nandu,true,2021-10-06T19:50:29.211Z,2021-10-06T20:15:04.093Z,true
```

I would be confused by this information. For starters its hard to identify what's what, a column, ... 
The `color`, properties like `online` and more are also irrelevant for the CSV exporters I guess. Maybe be we should discuss the format?

I think the properties of interest are the notes and you could map the column id's on the names.. maybe. I'm not sure